components/sidebar/sidebar_channel_button_or_link/sidebar_channel_button_or_link.jsx

```diff
@@ -44,6 +45,14 @@ export default class SidebarChannelButtonOrLink extends React.PureComponent {
 
     render = () => {
         let badge = null;
+        let usernamePlaceHolder = this.props.displayName;
+        if (!this.props.displayName) {
```

Comment:
Given the assignment of `usernamePlaceHolder` to `this.props.displayName` above, perhaps use same here?

Reply:
Yes that makes it more obvious